Cargo.toml

```diff
@@ -74,7 +74,7 @@ insta.opt-level = 3
 
 [profile.dev]
 # release level optimizations otherwise everything feels slow
-# opt-level = 3
+opt-level = 3
```

Comment:
we really aught to given @grahamking his own profile.  he wants this because cpu decoding is slow without it. 

trying to find the balance for compile and link time iterations.  of source more modularity would be beneficial here too.